docs/security/sso_idp_integration.md

```diff
@@ -0,0 +1,157 @@
+# SSO: Percona Everest IdP integration
+
+[Identity Provider (IdP)](https://www.okta.com/identity-101/why-your-company-needs-an-identity-provider/){:target="_blank"} integration connects applications and services with an external identity provider for your organization. This enables centralized authentication and authorization management, improving security and simplifying user access. By leveraging IdP integration, you can ensure that users are securely authenticated and authorized to access various applications and services across your organization.
```

Comment:
Is that expected that in this PR this file is added as new? It seems not to be directly related to this PR